### ISSUE TITLE
Is python-jose still supported?

### ISSUE THREAD

**Issue by jusdino** [mpdavis/python-jose#340]
I see there have been no releases in a couple of years and issues/pull requests stacking up in this repo.

Can anybody who still maintains this library comment on its level of support (if any), just so everybody understands?

**Comment by Jamim** [comment]
Hello @jusdino,

Thank you for raising this question!
There is a similar issue already:
  * #332

**Comment by jusdino** [comment]
Yep, saw that. I wanted to explicitly ask about any intent to support in general rather than just the next update. I see a lot of libraries out there still using this as a dependency, maintainers probably not realizing that this library appears unmaintained, rather than just not needing updates.

**Comment by zhu0629** [comment]
use [authlib](https://github.com/lepture/authlib) instead of python-jose
